external/HavoqGT/README.md

```diff
@@ -1,26 +1,31 @@
 
 # Required to Build HavoqGT with UMap
 
+We assume that the following items are already available (installed) on the system:
 - GCC 8.1 or more.
 - MPI
 - CMake 2.6 or more.
-- [Boost C++ Libraries](https://www.boost.org/) 1.64 or more.
-- [Metall](https://github.com/LLNL/metall)
+
+Since HavoqGT is not a read-only application,
+the write-protect support for userfaultfd() must be supported on the system.
 
 # Build
-One can install Boost C++ Libraries and Metall using Spack.
-A proper version of Boost C++ Libraries will be installed along with Metall.
+
+HaoqGT depends on Boost C++ Libraries and [Metall](https://github.com/LLNL/metall).
+One can install them and UMap using Spack.
 
 An example of building HavoqGT using Spack is:
 ```bash
 spack install umap
 spack load umap
 
+# A proper version of Boost C++ Libraries will be installed along with Metall.
```

Comment:
This comment isn't clear. Above it says HavoqGT depends on Boost (and please correct spelling of HavoqGT) and boost can be installed with Spack. Line 22 seems to say there is no need to install boost separately since it will be installed as part of the Metall install. Please clarify.